connector-email/pom.xml

```diff
@@ -75,6 +75,27 @@
             <artifactId>camel-quarkus-http</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-rest-client-reactive-jackson</artifactId>
```

Comment:
I'm not sure it's safe to use `quarkus-rest-client` from a Camel app. Maybe we should rely a Camel HTTP call instead.

Reply:
Right, let's refactor it using regular camel HTTP component